signature/policy_eval_signedby.go

```diff
@@ -47,20 +47,22 @@ func (pr *prSignedBy) isSignatureAuthorAccepted(image types.UnparsedImage, sig [
 		return sarRejected, nil, err
 	}
 	defer os.RemoveAll(dir)
-	mech, err := newGPGSigningMechanismInDirectory(dir)
-	if err != nil {
-		return sarRejected, nil, err
-	}
+	/*
+		mech, err := newGPGtypes.SigningMechanismInDirectory(dir)
+		if err != nil {
+			return sarRejected, nil, err
+		}
+	*/
```

Comment:
@mtrmac i think I'm fine with build tags for now although I will prefer the `containers_image_gpgme` to be the tag (in that way we don't have to add build tag to all build scripts in origin to make sure we don't build anything that requires openpgp).

We have to talk how I'm going to use this library in Origin as I think we are still on the same page (we just going to verify the signature, not policy.json, so probably we will just use the `signature` initially)